cmd/commands.go

```diff
@@ -203,6 +204,12 @@ var (
 		AddSSHKeyCommand,
 		RemoveSSHKeyCommand,
 
+		// Integrations
+		IntegrationsListCommand,
```

Comment:
All these 4 functions can be private, they are never used outside of this package